protected/humhub/modules/space/widgets/HeaderControlsMenu.php

```diff
@@ -6,7 +6,7 @@
  * @license https://www.humhub.com/licences
  */
 
-namespace humhub\modules\space\widgets;
```

Comment:
I don't see why this is being done within the space core module and not in the user core module instead? This would break an instance. Suggestion move these changes to the user core module. :smile:

Reply:
Changes were made to both the space and the user, but they don't break anything.

Reply:
Oh this is separate. This shouldn't be part of the PR. This is a different project where I'm trying to clone the space dropdown menu and add it to user profiles as well

Reply:
I removed it from the PR